### PR TITLE
LL-4372 (Stratis): eol warning banner added

### PR DESCRIPTION
**Pull request by LFBarreto**
(Stratis): eol warning banner added

localhost_8080_webpack_index html_theme=null

### Context

LL-4372
### Parts of the app affected / Test plan

Apps with Stratis accounts should see the warning message

### CODE REVIEW

**Review by juan-cortes — DISMISSED** [review]
"October 16th" or "16th of October" maybe?
cc @dasilvarosa

File: src/config/urls.js [hunk 1]
```diff
@@ -142,6 +142,8 @@ export const urls = {
       "https://support.ledger.com/hc/en-us/articles/360025321733?utm_source=ledger_live_desktop&utm_medium=self_referral&utm_content=error_wrongdevice",
     SyncError:
       "https://support.ledger.com/hc/en-us/articles/360012109220?utm_source=ledger_live_desktop&utm_medium=self_referral&utm_content=error_syncerror",
+    StratisDown2021Warning:
+      "https://support.ledger.com/hc/en-us/articles/115005175329-Stratis-STRAT-",
```
Comment (dasilvarosa) [inline src/config/urls.js:146]: zendesk uses Canonical links, no need for the title at the end. You can use:
https://support.ledger.com/hc/en-us/articles/115005175329

**Review by dasilvarosa — CHANGES_REQUESTED** [review]

**Review by gre — DISMISSED** [review]

**Review by nabil-brn — DISMISSED** [review]
✓ QA-ed

---
c7ed4ba8b (HEAD -> LL-4372, luiz/LL-4372) (Stratis): warning banner url updated

**Comment by gre** [comment]
Conflicts

**Review by gre — APPROVED** [review]

**Review by valpinkman — APPROVED** [review]

**Review by Arnaud97234 — APPROVED** [review]

**Review by dasilvarosa — APPROVED** [review]

**Pull request merged**